src/pages/VouchingPage/NewNominationModal.tsx

```diff
@@ -59,69 +52,209 @@ export const NewNominationModal = ({
-    if (!nominateChanged) {
-      throw 'Submit called when form not ready.';
-    }
+  const onSubmit: SubmitHandler<NominateFormValues> = async data => {
     nominateUser({
-      name,
-      address,
-      description,
+      ...data,
     })
       .then(() => {
         onClose();
       })
       .catch(console.warn);
   };
-
   return (
-    <FormModal
-      title="Nominate New Member"
-      submitDisabled={!nominateChanged}
-      onSubmit={onSubmit}
-      submitText="Nominate Member"
-      open={visible}
-      onClose={onClose}
-      size="small"
-    >
-      <p className={classes.description}>{nominateDescription}</p>
-      <div className={classes.quadGrid}>
-        <ApeTextField
-          label="Name"
-          value={name}
-          onChange={onChangeWith(setName)}
-          fullWidth
-        />
-        <ApeTextField
-          label="ETH Address"
-          value={address}
-          onChange={onChangeWith(setAddress)}
-          error={address.length > 0 && !isAddress}
-          fullWidth
-        />
-        <ApeTextField
-          label="Why are you nominating this person?"
-          placeholder="Tell us why the person should be added to the circle, such as what they have achieved or what they will do in the future."
-          value={description}
-          className={classes.gridAllColumns}
-          onChange={onChangeWith(setDescription)}
-          multiline
-          rows={4}
-          inputProps={{
-            maxLength: 280,
+    <Modal title="Nominate New Member" open={visible} onClose={onClose}>
+      <Form
+        css={{
+          position: 'relative',
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          backgroundColor: 'white',
+          width: '100%',
+          padding: '0 0 $lg',
+          overflowY: 'auto',
+          maxHeight: '100vh',
+        }}
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <Text css={{ lineHeight: 'normal' }}>{nominateDescription}</Text>
+        <Box
+          css={{
+            display: 'grid',
+            width: '100%',
+            'grid-template-columns': '1fr 1fr',
+            'grid-template-rows': 'auto auto',
+            'column-gap': '$lg',
+            'row-gap': '$xl',
           }}
-          fullWidth
-        />
-      </div>
-    </FormModal>
+        >
+          <Box
+            css={{
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+              width: '100%',
+            }}
+          >
+            <FormLabel
+              htmlFor="name"
+              css={{
+                lineHeight: '1.3',
+                color: '$text',
+                fontSize: '$4',
+                fontFamily: 'Inter',
+                fontWeight: '$bold',
+                textAlign: 'center',
+                m: '0 0 $sm',
+              }}
+            >
+              Name
+            </FormLabel>
+            <Controller
```

Comment:
Can you use the `useController` hook instead? It's a lot nicer IMO because the form logic is more separated from presentation, and also there's less nesting of components. Check out what we did in VaultsPage/CreateForm for an example.

Reply:
I updated it to use useController with zodresolver instead of passing "rules" to useController